### ISSUE TITLE
dRICH flowchart is out of date

### ISSUE THREAD

**Issue by c-dilks** [eic/EICrecon#1211]
**Is your feature request related to a problem? Please describe.**
The [dRICH flowchart](https://github.com/eic/EICrecon/blob/main/src/detectors/DRICH/README.md#flowchart) has fallen a bit behind, in particular, the parts relevant to tracking/trajectories.

**Describe the solution you'd like**
Make sure all algorithm inputs and outputs are correct, from:
- [ ] [DRICH.cc](https://github.com/eic/EICrecon/blob/main/src/detectors/DRICH/DRICH.cc); in particular, recent changes to track propagation inputs
- [ ] [pid.cc](https://github.com/eic/EICrecon/blob/main/src/global/pid/pid.cc); possibly also take into account https://github.com/eic/EICrecon/pull/1210

**Describe alternatives you've considered**
Use the `janadot` to generate this, but that typically gives more info than we need and is harder to read. We've also used this `mermaid` flowchart for planning and presentations, so it's useful to keep it around.

**Additional context**
Any changes to the dRICH or PID algorithm inputs or outputs should require documentation updates.

CC. @chchatte92 @bleaktwig

**Comment by chchatte92** [comment]
@c-dilks shall this part to be updated?
https://github.com/eic/EICrecon/blob/5bf3d29f315d5df09ef463448443b106ff979456/src/detectors/DRICH/DRICH.cc#L112

**Comment by c-dilks** [comment]
Yes, but it would be good to also cross check the others. You can also check the git blame for recent changes (upper-left corner, click 'Blame').

`mermaid` is used for these flowcharts; in case anyone needs it, here's some documentation: https://mermaid.js.org/syntax/flowchart.html
